Update Python logging examples in documentation.

cgoldberg opened the pull request:
### **User description**
### Description
This adds information to the logging docs in `documentation/webdriver/troubleshooting/` for setting up log handlers in Python.

Fixes [SeleniumHQ/selenium #15359](https://github.com/SeleniumHQ/selenium/issues/15359)

### Motivation and Context

The examples were incomplete and this is necessary to show how to get log output in Python.

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [X] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->


___

### **PR Type**
Documentation


___

### **Description**
- Added Python logging handler examples to documentation.

- Updated logging examples in English, Japanese, Portuguese, and Chinese.

- Improved clarity on setting up Python log handlers.


___



### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>logging.en.md</strong><dd><code>Add Python logging handler example in English documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/troubleshooting/logging.en.md

<li>Added example for setting up a Python logging handler.<br> <li> Included <code>StreamHandler</code> and <code>FileHandler</code> usage for console logs.


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2206/files#diff-1513ff59adee30b4bc0a99719539643e8426a0f6ab8ec7ee059eb46fd4034bab">+6/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>logging.ja.md</strong><dd><code>Add Python logging handler example in Japanese documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/troubleshooting/logging.ja.md

<li>Added example for setting up a Python logging handler.<br> <li> Included <code>StreamHandler</code> and <code>FileHandler</code> usage for console logs.


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2206/files#diff-3a3b25a149ff8f29520896832bbd87f4cc9f4d2330c2753f5c7a278b9f358a83">+6/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>logging.pt-br.md</strong><dd><code>Add Python logging handler example in Portuguese documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/troubleshooting/logging.pt-br.md

<li>Added example for setting up a Python logging handler.<br> <li> Included <code>StreamHandler</code> and <code>FileHandler</code> usage for console logs.


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2206/files#diff-d09af2c5582bf262a58e57d0a19269bd9483af0a4aff035edecc55067d80f876">+6/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>logging.zh-cn.md</strong><dd><code>Add Python logging handler example in Chinese documentation</code></dd></summary>
<hr>

website_and_docs/content/documentation/webdriver/troubleshooting/logging.zh-cn.md

<li>Added example for setting up a Python logging handler.<br> <li> Included <code>StreamHandler</code> and <code>FileHandler</code> usage for console logs.


</details>


  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/2206/files#diff-070d45dfd2ac72540adb0ba20f6bb1029a0a93c8d5bc8e4ab2372c5840df1139">+6/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> <details> <summary>  Need help?</summary><li>Type <code>/help how to ...</code> in the comments thread for any questions about Qodo Merge usage.</li><li>Check out the <a href="https://qodo-merge-docs.qodo.ai/usage-guide/">documentation</a> for more information.</li></details>

cgoldberg committed:
Update Python logging examples in documentation.

This adds information to the logging docs in
`documentation/webdriver/troubleshooting/`` for setting up
log handlers in Python. This is necessary to get log output.

harsha509 committed:
Merge branch 'trunk' into python-log-examples

Delta456 commented on website_and_docs/content/documentation/webdriver/troubleshooting/logging.en.md:
````diff
@@ -27,6 +27,12 @@ take a look at the [Selenium Logger project](https://github.com/titusfortner/sel
 {{< gh-codeblock path="/examples/python/tests/troubleshooting/test_logging.py#L5" >}}
+You must also set a handler (`StreamHandler`, `FileHandler('/path/to/log')`, etc).
+For example, to see logs in the console, you can do this:
+```py
+handler = logging.StreamHandler()
+logger.addHandler(handler)
+```
````
How about simply referring to https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk//examples/python/tests/troubleshooting/test_logging.py#L9-L10 as a code block?

cgoldberg replied:
That's an option too. The changes I added show the `StreamHandler` for getting console logging, which I think is a more common use case. I guess it would be good to show both, so users see how to do file and console logging. I'll update the PR to show both.

cgoldberg committed:
Merge branch 'SeleniumHQ:trunk' into python-log-examples

cgoldberg committed:
Add file logging to Python logging examples

cgoldberg commented:
@Delta456 
I updated my branch so it shows examples of both `StreamHandler` and `FileHandler` for console and file logging. Rather than showing the code block from `test_logging.py` like you suggested, I added the examples directly to the document... because the example in the test file gets `log_path` from a test fixture, which I thought might confuse users. My example shows explicitly setting the `log_path` value.

Let me know what you think.

Delta456 commented:
> @Delta456 I updated my branch so it shows examples of both `StreamHandler` and `FileHandler` for console and file logging. Rather than showing the code block from `test_logging.py` like you suggested, I added the examples directly to the document... because the example in the test file gets `log_path` from a test fixture, which I thought might confuse users. My example shows explicitly setting the `log_path` value.
> 
> Let me know what you think.

This is a better way than what I originally proposed. Thanks!

cgoldberg committed:
Merge branch 'SeleniumHQ:trunk' into python-log-examples

cgoldberg commented:
@Delta456 Do you have access to merge this? (I don't). If not, who needs to pull the trigger to land this in trunk?

Delta456 commented:
> @Delta456 Do you have access to merge this? (I don't). If not, who needs to pull the trigger to land this in trunk? 
> 

I also don't have access to merge this. @harsha509 or any other TLC member can merge this.

harsha509 reviewed (approved):
Thank you @cgoldberg !
